src/components/CreateRequest/Details/index.js

```diff
@@ -150,6 +151,15 @@ class Details extends Component {
       return;
     }
 
+    const documentURI = this.state.documentURI;
+    const requestTrainingDSURI = this.state.requestTrainingDS;
```

Comment:
you can detructure the state. It will be in one line.

Reply:
Will add it in the next release. Used this options as it is only two variables to Validate.